### PR TITLE
Add new cask 'dvc'

### PR DESCRIPTION
**Pull request by ivaquero**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by ivaquero** [comment]
This script currently need the uninstall stanza that I am not sure what to put here. And I am getting in touch with dvc official developers for help.

File: Casks/dvc.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cask "dvc" do
+  version "2.33.2"
+  sha256 "ceebcb6be244f6fdb5c6329ccd0ceb2cc25c727c6a4929f7d690716a708b90af"
+
+  # github.com/iterative/dvc/ was verified as official when first introduced to the cask
+  url "https://github.com/iterative/dvc/releases/download/#{version}/dvc-#{version}.pkg"
```
Comment (efiop) [inline Casks/dvc.rb:6]: github releases are not official packages and are unsigned.
```suggestion
  url "https://dvc.org/download/osx/dvc-#{version}.pkg"
```

File: Casks/dvc.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
+  depends_on macos: ">= :mojave"
+
+  pkg "dvc-#{version}.pkg"
+
+  zap trash: [
```
Comment (efiop) [inline Casks/dvc.rb:20]: i guess you are missing
```suggestion
  pkg "dvc-#{version}.pkg"

  uninstall pkgutil: 'com.iterative.dvc'

  zap trash: [
```
but not 100% sure

**Comment by bevanjkay** [comment]
@ivaquero Thank you for your work in preparing this, but it looks like `dvc` already exists in `homebrew-core`. 
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/dvc.rb

**Comment by MikeMcQuaid** [comment]
> Yeah, I saw the package in Homebrew Core. However, a Cask takes less time to download.

Sorry, that's not a good enough reason.

**Comment by ivaquero** [comment]
I understand your consideration, while I have more than one reason. 

Anyway, I am ready to accept the decision from any maintainers.

**Pull request closed without merging**